test/onnx/test_pytorch_onnx_caffe2.py

```diff
@@ -1473,9 +1473,16 @@ def forward(self, x):
         x = torch.randn(16, 3, 256, 256)
         self.run_model_test(ReduceSumMultipleAxes(), train=False, input=(x,), batch_size=BATCH_SIZE, use_gpu=False)
 
+    @skipIfEmbed
```

Comment:
why is this needed?

Reply:
https://github.com/pytorch/pytorch/blob/7e2a5c34e31cbbc932a59d47c9860b42f88672dc/test/onnx/test_pytorch_onnx_caffe2.py#L1231
Apparently instance norm includes unused weights.